src/common/datetime/format_time.ts

```diff
@@ -17,3 +17,12 @@ export const formatTimeWithSeconds = toLocaleTimeStringSupportsOptions
         second: "2-digit",
       })
   : (dateObj: Date) => format(dateObj, "mediumTime");
+
+export const formatTimeWeekday = toLocaleTimeStringSupportsOptions
+  ? (dateObj: Date, locales: string) =>
+      dateObj.toLocaleTimeString(locales, {
+        weekday: "long",
+        hour: "numeric",
+        minute: "2-digit",
+      })
+  : (dateObj: Date) => format(dateObj, "shortTime");
```

Comment:
This is not correct, check the right format in the docs of fecha <https://github.com/taylorhakes/fecha#formatting-tokens>

Reply:
Not sure how I missed that 🤦 ... fixed now.